From 04f01a61ba2451f474be8a32586a55960b852248 Mon Sep 17 00:00:00 2001
From: Thomas Weber <muffin@muffin.ink>
Date: Sun, 5 Jul 2026 12:39:18 -0500
Subject: Ignore non-release tags in get-git-version.py

Our `dangotron-base` tag was causing the script to return an version
that is not semver and causes errors during the build.
---
 script/get-git-version.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/script/get-git-version.py b/script/get-git-version.py
index 8b80353203..09dc89bf25 100755
--- a/script/get-git-version.py
+++ b/script/get-git-version.py
@@ -21,7 +21,8 @@ import re
 
 try:
   output = subprocess.check_output(
-      ['git', 'describe', '--tags', '--abbrev=0'],
+      # Dangotron: ignore the non-vX.Y.Z tags that we use for our patch management
+      ['git', 'describe', '--tags', '--abbrev=0', '--match', 'v*'],
       cwd=os.path.abspath(os.path.join(os.path.dirname(__file__), '..')),
       stderr=subprocess.PIPE,
       universal_newlines=True)
-- 
2.55.0

